docs/protocol/terminology.mdx

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 2
+title: "Terminology"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Terminology
+
+## Core Concepts
+
+**Channel**: A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.
+
+**State**: A snapshot of the channel at a point in time, including fund allocations and application-specific data.
+
+**Participant**: An entity (identified by a wallet address) that is part of a channel. Typically includes a Creator (user) and a Clearnode or Clearnode Operator.
+
+**Clearnode**: A virtual ledger layer that provides a unified ledger (through Nitro RPC) and coordinates state channels (through Nitrolite), providing chain abstraction for developers and users.
+
+**Creator**: The participant at index 0 in a channel who initiates channel creation. The Creator constructs the channel configuration, prepares the initial funding state, signs it (signature at position 0), and calls the on-chain `create()` function to lock their funds and establish the channel. Typically a user or light client opening a payment channel with a clearnode.
+
+**App Sessions**: Off-chain channels built on top of payment channels, intended to be used by app developers to enable application-specific interactions and transactions without touching the blockchain.
+
+**Unified Balance**: An abstraction that aggregates a user's funds across multiple blockchain networks, managed by a clearnode.
+
+**Session Key**: A temporary cryptographic key delegated by a user's main wallet that provides a flexible way for the user to manage security of their funds by giving specific permissions and allowances for specific apps.
+
+## Identifiers
+
+**channelId**: A unique identifier for a channel, computed as the hash of the channel configuration.
+
+**stateHash**: A cryptographic hash of a channel state, used for signature verification.
+
+**requestId**: A unique identifier for an RPC request, used for correlating requests and responses.
+
+**appSessionId**: A unique identifier for an app session, formatted as a 0x-prefixed hex string (32 bytes). Used for all subsequent operations on that specific app session.
```

Comment:
> ... formatted as a 0x-prefixed hex string (32 bytes).

I think that we should either omit this or add the same for "channelId", "stateHash", "requestId" and "appSessionId".